src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private int maxFig = 6;
+    private int maxBound = 50;
+
```

Comment:
Если ты хочешь задавать параметры для рандома извне, тогда лучше убрать инициализацию по умолчанию, и добавить эти поля в конструктор
```suggestion
    private int maxFig;
    private int maxBound;
    
    public FigureSupplier(int maxFig, int maxBound) {
        this.maxFig = maxFig;
        this.maxBound = maxBound;
    }
```